app/views/tag/_sorting.html.erb

```diff
@@ -14,7 +14,8 @@
         items: 8,
         minLength: 3,
         source: function (query, process) {
-          return $.post('/tag/suggested/' + query, {}, function (data) {
+          var replaced_query = query.replace(' ', /-/g);
+          return $.post('/tag/suggested/' + replaced_query, {}, function (data) {
             return process(data)
```

Comment:
Are we missing a semicolon here?

Reply:
On line 19? I did not change that part of the code, but I tested here with the semicolon and the result is the same. I'm almost sure that in javascript the semicolon isn't needed when you have only one statement in a block.

Reply:
Yes, you're right, it's not necessary, but it's not good practice. However, since you have confirmed that adding it makes no difference and that's not code you've written, let's leave it at that.
Regarding the solution, I think we'll have to wait for @jywarren to debug this. (I'm not very comfortable with JS anyway.)

Reply:
Thanks for your help! :)